tests/unit/s2n_pq_kem_kyber_available_awslc_test.c

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include "crypto/s2n_openssl.h"
+#include "pq-crypto/s2n_pq.h"
+#include "s2n_test.h"
+
+int main()
+{
```

Comment:
This might not need a whole new test file. Would it make sense in one of the existing test files?

Reply:
sure, i'll look into which one it might fit best.